crates/router/src/core/payments/operations/payment_approve.rs

```diff
@@ -181,6 +181,7 @@ impl<F: Send + Clone> GetTracker<F, PaymentData<F>, api::PaymentsCaptureRequest>
             authentication: None,
             recurring_details: None,
             poll_config: None,
+            is_payment_processor_token_flow: None,
```

Comment:
we do not need this field in payment data as we already have payment intent which has this field, use that instead